ui/app/AppLayouts/Wallet/panels/RecipientSelectorPanel.qml

```diff
@@ -1,42 +1,99 @@
+import AppLayouts.Wallet.controls 1.0
+import AppLayouts.Wallet.views 1.0
+
 import QtQuick 2.15
 import QtQuick.Layouts 1.15
-import QtQml.Models 2.1
 
+import StatusQ 0.1
+import StatusQ.Controls 0.1
 import StatusQ.Core 0.1
 import StatusQ.Core.Theme 0.1
 import StatusQ.Core.Utils 0.1 as StatusQUtils
-import StatusQ.Controls 0.1
-import StatusQ.Components 0.1
 
-import shared.controls 1.0 as SharedControls
-// TODO: remove all files and dependencies with this location once old send modal is removed
-import shared.popups.send.controls 1.0
-import shared.popups.send 1.0
+import shared.panels 1.0 as SharedPanels
 
 import utils 1.0
 
-import AppLayouts.Wallet.views 1.0
-
 Rectangle {
     id: root
 
-    required property var savedAddressesModel
-    required property var myAccountsModel
-    required property var recentRecipientsModel
+    /**
+        Expected roles (for both models):
+
+        address     [string] - unique wallet address
+        name        [string] (optional)
+        color       [string] (optional)
+        colorId     [string] (optional)
+        emoji       [string] (optional)
+        ens         [string] (optional)
+    */
+    required property var recipientsModel
+    required property var recipientsFilterModel
 
-    property alias selectedRecipientAddress: recipientInputLoader.selectedRecipientAddress
-    property alias selectedRecipientType: recipientInputLoader.selectedRecipientType
+    readonly property string searchPattern: recipientInputLoader.searchPattern
```

Comment:
this property is not used outside this component and can be made private or not needed at all?